packages/aws-client/src/StepFunctions.ts

```diff
@@ -28,15 +42,14 @@ export const doesExecutionExist = (describeExecutionPromise: Promise<unknown>) =
  * If a ThrottlingException is received, this function will retry using an
  * exponential backoff.
  *
- * @param {Object} params
- * @returns {Promise<Object>}
+ * @param {DescribeExecutionInput} params
+ * @returns {Promise<DescribeExecutionOutput>} DescribeExecutionOutput
```

Comment:
nit: as long as you're being helpful in describing the actual return type, a description would be excellent.

e.g.
`@returns {Promise<DescribeExecutionOutput>} An object representing a step function name and id` (or whatever it actually is)

We don't own `DescribeExecutionOutput` so explaining it is arguably not necessary and you could just do 

`@returns {Promise<DescribeExecutionOutput>}`

Obviously super minor and in no way a blocking comment 😄

Reply:
noted, I removed the type name from the return description